blueocean-rest-impl/src/test/java/io/jenkins/blueocean/service/embedded/ProfileApiTest.java

```diff
@@ -222,50 +222,27 @@ public void createUserFavouriteFolderTest() throws Exception {
         Assert.assertEquals(0, l.size());
 
 
-        map = new RequestBuilder(baseUrl)
+        new RequestBuilder(baseUrl)
             .put("/organizations/jenkins/pipelines/folder1/favorite/")
             .jwtToken(token)
             .data(ImmutableMap.of("favorite", true))
+            .status(405)
             .build(Map.class);
 
-        validateFolder(folder1, (Map) map.get("item"));
-        l = new RequestBuilder(baseUrl)
-            .get("/users/"+user.getId()+"/favorites/")
-            .jwtToken(token)
-            .build(List.class);
-
-        Assert.assertEquals(1, l.size());
-        Map folder = (Map)((Map)l.get(0)).get("item");
-
-        validateFolder(folder1, folder);
-
-        href = getHrefFromLinks((Map)l.get(0),"self");
-
-        Assert.assertEquals("/blue/rest/organizations/jenkins/pipelines/folder1/favorite/", href);
-
-        map = new RequestBuilder(baseUrl)
-            .put(href.substring("/blue/rest".length()))
```

Comment:
It does test that you can expect to favourite a folder and have it work. AFAIK there are no tests that do that.